R/tabulate_gee.R

```diff
@@ -4,14 +4,15 @@
 #' [fit_gee()].
 #'
 #' @name tabulate_gee
+#' @noRd
```

Comment:
Hi @edelarua , just want to double check why this is dropped. Thanks.

Reply:
Hi @shajoezhu, to better align with `tern`/`tern.rbmi` I separated out the statistics/formatted analysis/analyze functions into their own sections of the pkgdown reference so `tabulate_gee` no longer contains any functions.